publication-commons/src/main/java/no/unit/nva/publication/permissions/file/FileStrategyBase.java

```diff
@@ -65,8 +65,16 @@ protected boolean hasTopLevelRelationForCurrentResource() {
         return resource.getCuratingInstitutions().stream().anyMatch(org -> org.id().equals(userTopLevelOrg));
     }
 
+    private boolean hasCuratorRelation() {
+        return hasTopLevelRelationForResourceContributors() || hasTopLevelRelationForResourceOwner();
```

Comment:
this is the fix, other stuff is just a try to make tests more explicit

Reply:
This method check relation only, not "curator" relation, maybe rename it to something similar we have in PublicationPermissionStragies?

This is how exactly the same method is named there:

```
    protected boolean userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitution() {
        return userIsFromSameInstitutionAsPublicationOwner() || userBelongsToCuratingInstitution();
    }
```